src/flags.h

```diff
@@ -79,58 +79,58 @@
 #define CMD_TABLE \
 /* parent keys  */\
 /*  with touch  */\
-X(NO_CMD)          /* placeholder - keep first */\
-X(sign)           \
-X(seed)           \
-X(reset)          \
-X(password)       \
-X(bootloader)     \
-X(hidden_password)\
-X(REQUIRE_TOUCH)   /* placeholder - do not move */\
+X(NO_CMD, 0)          /* placeholder - keep first */\
+X(sign, 10)           \
+X(seed, 11)           \
+X(reset, 12)          \
+X(password, 13)       \
+X(bootloader, 14)     \
+X(hidden_password, 15)\
+X(REQUIRE_TOUCH, 0)   /* placeholder - do not move */\
 /* parent keys  */\
 /*  w/o touch   */\
-X(verifypass)     \
-X(led)            \
-X(xpub)           \
-X(name)           \
-X(ecdh)           \
-X(device)         \
-X(random)         \
-X(backup)         \
-X(aes256cbc)      \
-X(ping)           \
+X(verifypass, 16)     \
```

Comment:
Maybe leave a gap in numbers here, in case we want to add more commands that require touch button press.